src/Notepads/App.xaml.cs

```diff
@@ -168,6 +169,16 @@ private async Task ActivateAsync(IActivatedEventArgs e)
         private Frame CreateRootFrame(IActivatedEventArgs e)
         {
             Frame rootFrame = new Frame();
+
+            var flowDirectionSetting = ResourceContext.GetForCurrentView().QualifierValues["LayoutDirection"];
```

Comment:
This seems hacky, are you sure this is the correct or proper way of detecting RTL?

Reply:
Yes, I found it from [documentation](https://docs.microsoft.com/en-us/uwp/api/windows.applicationmodel.resources.core.resourcecontext.QualifierValues?view=winrt-19041) and tested it. This is the only way we can fix this bug until MS [fixes](https://github.com/microsoft/microsoft-ui-xaml/issues/4891) it themselves.